.github/workflows/unit.yaml

```diff
@@ -14,9 +14,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-go@v3
+      - uses: actions/setup-go@v4
         with:
           go-version-file: "go.mod"
+      - id: cache-vendor
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-vendor-dir
+        with:
+          path: ./vendor
+          key: go-vendor-${{ hashFiles('go.sum') }}
+          enableCrossOsArchive: true
+      - if: ${{ steps.cache-vendor.outputs.cache-hit != 'true' }}
+        name: Run make vendor
+        run: make vendor
```

Comment:
Is this just a weird display or are L30, L31 a nesting problem?

Reply:
It just looks weird since the `make vendor` step has the `-` up above on line 28. It would probably look better if it was:
```yaml
      - name: Run make vendor
        run: make vendor
        if: ${{ steps.cache-vendor.outputs.cache-hit != 'true' }}
      - run: make unit
```